src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.props

```diff
@@ -8,9 +8,8 @@
     <MinimumSupportedJavaVersion Condition=" '$(MinimumSupportedJavaVersion)' == '' ">1.8.0</MinimumSupportedJavaVersion>
     <EnableDefaultOutputPaths Condition=" '$(EnableDefaultOutputPaths)' == '' And '$(OS)' != 'Windows_NT' ">false</EnableDefaultOutputPaths>
     <EnableDefaultOutputPaths Condition=" '$(EnableDefaultOutputPaths)' == '' ">true</EnableDefaultOutputPaths>
-    <DisableImplicitAndroidFrameworkReference Condition=" '$(DisableImplicitAndroidFrameworkReference)' == '' ">false</DisableImplicitAndroidFrameworkReference>
-    <EnableDefaultAndroidResourceItems Condition=" '$(EnableDefaultAndroidResourceItems)' == '' ">true</EnableDefaultAndroidResourceItems>
-    <EnableDefaultAndroidAssetItems Condition=" '$(EnableDefaultAndroidAssetItems)' == '' ">true</EnableDefaultAndroidAssetItems>
+    <EnableDefaultItems Condition=" '$(EnableDefaultItems)' == '' ">true</EnableDefaultItems>
```

Comment:
Isn't this set automatically by `Microsoft.NET.Sdk`?

Reply:
It should be, and we should always be imported after, so it should be fine to remove this.

Reply:
Actually it's set in `Microsoft.NET.Sdk.DefaultItems.targets`: https://github.com/dotnet/sdk/blob/32496b832d0e7702a0dca105160d97bf0c84c7ae/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L16, which means I don't think it'll be set in `Microsoft.Android.Sdk.props`.

I moved the default inclusion logic to our .targets file because of this: https://github.com/rolfbjarne/xamarin-macios/commit/84e62e117f4bdfa6768b3e50952c7017e6e1673c